README.md

```diff
@@ -166,7 +166,7 @@ When defining Consumers the items below make up the entire configuration of the
 The rest of this document introduces the `nats` utility, but for completeness and reference this is how you'd create the ORDERS scenario.  We'll configure a 1 year retention for order related messages:
 
 ```bash
-$ nats str add ORDERS --subjects "ORDERS.*" --ack --max-msgs=-1 --max-bytes=-1 --max-age=1y --storage file --retention limits --max-msg-size=-1 --discard=old
+$ nats str add ORDERS --subjects=ORDERS.* --ack --max-msgs=-1 --max-bytes=-1 --max-age=1y --storage=file --retention=limits --max-msg-size=-1 --discard=old
```

Comment:
this wont work, subjects need to be quoted if you have `*` or `>`

Reply:
if you put the = after the subject it works ok, putting the quote actually stores "ORDERS.*" as a subject which doesn't work

Reply:
What shell do you use? It's broken :)

Reply:
```
$ nats str add ORDERS --subjects="ORDERS.*" --ack --max-msgs=-1 --max-bytes=-1 --max-age=1y --storage file --retention limits --max-msg-size=-1 --discard=old
$ nats str info ORDERS -j |jq .config.subjects
[
  "ORDERS.*"
]
```

Reply:
Also the --discard flag is not part of the command.

Reply:
It is in master?

Reply:
I am using your jsm:latest

Reply:
Yeah it's lagging a bit, no real work has been happening on JS for a while.  try jsm:nightly

Reply:
weird:

```
Flags:
  -h, --help                     Show context-sensitive help (also try --help-long and --help-man).
      --version                  Show application version.
  -s, --server="localhost:4222"  NATS servers
      --user=USER                Username of Token
      --password=PASSWORD        Password
      --creds=CREDS              User credentials
      --tlscert=TLSCERT          TLS public certificate
      --tlskey=TLSKEY            TLS private key
      --tlsca=TLSCA              TLS certificate authority chain
      --timeout=2s               Time to wait on responses from NATS
      --config=CONFIG            JSON file to read configuration from
      --subjects=SUBJECTS ...    Subjects that are consumed by the Stream
      --ack                      Acknowledge publishes
      --max-msgs=0               Maximum amount of messages to keep
      --max-bytes=MAX-BYTES      Maximum bytes to keep
      --max-age=""               Maximum age of messages to keep
      --storage=STORAGE          Storage backend to use (file, memory)
      --retention=RETENTION      Defines a retention policy (limits, interest, work)
      --max-msg-size=MAX-MSG-SIZE  
                                 Maximum size any 1 message may be
  -j, --json                     Produce JSON output
```

```
nats stream add ORDERS --ack --max-msgs=-1 --max-bytes=-1 --max-age=1y --storage=file --retention=limits --
max-msg-size=-1 --subjects="orders.*" --discard=old
nats: error: unknown long flag '--discard', try --help
```

Reply:
0.0.14 is the version reported.

Reply:
using zsh